march_shared_classes/test/subgait_test.py

```diff
@@ -35,6 +35,19 @@ def test_from_file_none_path(self):
     def test_from_file_no_robot(self):
         self.assertIsNone(Subgait.from_file(None, self.subgait_path))
 
+    # Subgait.from_files_interpolated tests
+    def test_from_files_interpolated_correct(self):
+        base_file_name = '{rsc}/{gait}/{subgait}/{version}.subgait'.format(rsc=self.resources_folder,
+                                                                           gait=self.gait_name,
+                                                                           subgait='left_close',
+                                                                           version='MV_walk_leftclose_v1')
+        other_file_name = '{rsc}/{gait}/{subgait}/{version}.subgait'.format(rsc=self.resources_folder,
```

Comment:
```suggestion
        base_subgait_path = '{rsc}/{gait}/{subgait}/{version}.subgait'.format(rsc=self.resources_folder,
                                                                           gait=self.gait_name,
                                                                           subgait='left_close',
                                                                           version='MV_walk_leftclose_v1')
        other_subgait_path = '{rsc}/{gait}/{subgait}/{version}.subgait'.format(rsc=self.resources_folder,
```
Your naming is not consistent with the method below ;)